tests/override_config_table/utilities.py

```diff
@@ -32,6 +32,8 @@ def reload_minigraph_with_golden_config(duthost, json_data, safe_reload=True):
     golden_config = "/etc/sonic/golden_config_db.json"
     duthost.copy(content=json.dumps(json_data, indent=4), dest=golden_config)
     config_reload(duthost, config_source="minigraph", safe_reload=safe_reload, override_config=True)
+    # Cleanup golden config because some other test or device recover may reload config with golden config
+    duthost.command('mv {} {}_backup'.format(golden_config, golden_config))
```

Comment:
Would it be better if we backup golden_config_db.json before the test and restore it after the case is finished like config_db.json?

Reply:
As my understand, today the golden_config_db.json should never exist, right?
The reason golden_config_db.json exist are:
1. someone left a golden_config_db.json on device, in this case it's a human error.
2. some test case left a golden_config_db.json, just like what happened now, it's a test case error.

So not restore not break anything and bring new issue.

Reply:
Thanks, approved.